config/etcd-io/org.yaml

```diff
@@ -0,0 +1,138 @@
+admins:
+- ahrtr
+- caniszczyk
+- cblecker
+- k8s-ci-robot
+- k8s-github-robot
+- MadhavJivrajani
+- mitake
+- mrbobbytables
+- nikhita
+- palnabarun
+- Priyankasaggu11929
+- ptabor
+- serathius
+- spzala
+billing_email: etcd-maintainers@googlegroups.com
+default_repository_permission: read
+description: etcd Development and Communities
+email: etcd-dev@googlegroups.com
+has_organization_projects: false
+has_repository_projects: true
+members:
+- cenkalti
+- chalin
+- chaochn47
+- eduartua
+- fanminshi
+- fuweid
+- idvoretskyi
+- jmhbnz
+- lavacat
+- lburgazzoli
+- nate-double-u
+- pavelkalinnikov
+- savitharaghunathan
+- tbg
+- tjungblu
+- victortrac
+- vorburger
+- wenjiaswe
+members_can_create_repositories: false
+name: etcd-io
+teams:
+  maintainers-bbolt:
+    description: Granted write access to bbolt
+    maintainers:
+    - ahrtr
+    - ptabor
+    - serathius
+    privacy: closed
+    repos:
+      bbolt: maintain
+  maintainers-discovery:
+    description: discovery.etcd.io maintainers
+    members:
+    - eduartua
+    - idvoretskyi
+    - victortrac
+    privacy: closed
+    repos:
+      discovery.etcd.io: maintain
+      discoveryserver: maintain
+  maintainers-etcd:
+    description: Granted write access to etcd
+    maintainers:
+    - ahrtr
+    - mitake
+    - ptabor
+    - serathius
+    - spzala
+    members:
+    - wenjiaswe
+    privacy: closed
+    repos:
+      dbtester: maintain
+      etcd: maintain
+      gofail: maintain
+  maintainers-jetcd:
+    description: Granted write access to jetcd
+    members:
+    - fanminshi
+    - lburgazzoli
+    - vorburger
+    privacy: closed
+    repos:
+      jetcd: maintain
+  maintainers-raft:
+    description: raft maintainers
+    maintainers:
+    - ahrtr
+    - mitake
+    - ptabor
+    - serathius
+    - spzala
+    members:
+    - tbg
+    privacy: closed
+    repos:
+      raft: maintain
+  maintainers-website:
+    description: ""
+    maintainers:
+    - ahrtr
+    - mitake
+    - ptabor
+    - serathius
+    - spzala
+    members:
+    - chalin
+    - nate-double-u
+    - wenjiaswe
+    privacy: closed
+    repos:
+      website: admin
+  members:
+    description: ""
+    members:
+    - cenkalti
+    - chaochn47
+    - lavacat
+    - pavelkalinnikov
+    - tjungblu
+    privacy: closed
+    repos:
+      bbolt: triage
+      dbtester: triage
+      etcd: triage
+      gofail: triage
+      raft: triage
+      website: triage
+  reviewers-etcd:
+    description: ""
+    members:
+    - fuweid
+    - jmhbnz
+    privacy: closed
+    repos:
+      etcd: triage
```

Comment:
Need to double check these `reviewers-etcd` permissions reflect a recent update https://github.com/etcd-io/website/issues/730.

Reply:
+1